src/bundle/Resources/public/scss/ui/modules/content-tree/_header.scss

```diff
@@ -1,21 +1,23 @@
 .c-header {
     display: flex;
     align-items: center;
+    justify-content: space-between;
     margin-bottom: calculateRem(8px);
-    padding: 0 calculateRem(4px) calculateRem(8px);
+    padding-bottom: calculateRem(8px);
     border-bottom: calculateRem(1px) solid $ibexa-color-light-500;
 
-    &__expand-btn {
-        display: flex;
+    &__expand-btn,
+    &__collapse-btn {
```

Comment:
Are these 100% the same? Because if they are we do not need 2 Elements. And if they are not, we probably need a modifier instead of another element.

Reply:
We don't need two classes so I changed the button class to `c-header__toggle-btn` because this class fits better